WordPress/gutenberg
# Store screenshots of CI e2e failures as CI artifacts

**Description**

This PR updates e2e testing workflow to ensure test failures are captured on screenshots for easier debugging.

**Testing instructions**

This PR purposefully breaks a test to trigger screenshot capture. To see if it works, go to e2e test failures and confirm you can see the following artifact:

<img width="1225" alt="Zrzut ekranu 2020-11-10 o 22 21 11" src="https://user-images.githubusercontent.com/205419/98735144-36be1380-23a3-11eb-9a10-f262c46eeb15.png">
